src/actions/transactionActions.ts

```diff
@@ -18,17 +18,20 @@ import { TransactionStatusType, UpdatedTransactionStatusType } from '../types/tr
 
 enum TransactionActionTypes {
   RESET_ESTIMATED_FEE = 'RESET_ESTIMATED_FEE',
-  CLEAR_ESTIMATED_FEE = 'CLEAR_ESTIMATED_FEE',
   SET_RECEIVER_ADDRESS = 'SET_RECEIVER_ADDRESS',
+  SET_ESTIMATED_FEE = 'SET_ESTIMATED_FEE',
```

Comment:
Why do we have `set` and `reset` now? Is there a difference between doing `setEstimatedFee(null)` and `resetEstimatedFee`? Introducing this kind of ambiguity is the reason why the code is perceived as "difficult" (note that we already had a very confusing `RESET/CLEAR` ambiguity earlier.

Reply:
i disagree on this for a couple of reasons:

1) i think that term `RESET` is quite descriptive about what the action is doing.. something that for example `setEstimatedFee(null)` is not so since you need to understand that estimatedFee is null hence is reseted.
2) there are some cases where the payload is an object with a couple of properties or a few parameters so is much cleaner to have a `resetSomething()` without any needed parameters than `setSomething(param1,param2,param3)`

Reply:
We don't anymore .  I have just followed your recommendation about doing: `setEstimatedFee(null,null)`

Reply:
Anyway i have created a new reset() action which is resetting all the corresponding states.